xml/Makefile.am

```diff
@@ -33,7 +33,7 @@ noinst_DATA		= context-of.xsl
 
 # Sorted list of available numeric RNG versions,
 # extracted from filenames like NAME-MAJOR[.MINOR][.MINOR-MINOR].rng
-RNG_numeric_versions    = $(shell ls -1 $(top_srcdir)/xml/*.rng \
+RNG_numeric_versions    = $(shell echo ${RNG_files} | tr ' ' '\n' \
```

Comment:
could as well be `for f in ${RNG_FILES}; do echo $$f; done | ....`, with
one less spawned process as a result